packages/front-end/components/Features/SDKConnections/SDKConnectionForm.tsx

```diff
@@ -739,36 +786,33 @@ export default function SDKConnectionForm({
                       </div>
                     </div>
                     {gb?.isOn("remote-evaluation") &&
-                    selectedLanguagesWithoutRemoteEvalSupport.length > 0 ? (
+                    !currentSdkCapabilities.includes("remoteEval") ? (
                       <div
-                        className="ml-2 mt-3 text-warning-orange small"
+                        className="ml-2 mt-3 text-warning-orange"
                         style={{ marginBottom: -5 }}
                       >
-                        <FaExclamationCircle /> Remote evaluation is currently
-                        only supported in a subset of front-end SDKs. It is not
-                        supported in the selected SDK
-                        {selectedLanguagesWithoutRemoteEvalSupport.length === 1
-                          ? ""
-                          : "s"}
-                        :
-                        <div className="ml-2 mt-1">
-                          {selectedLanguagesWithoutRemoteEvalSupport.map(
-                            (id, i) => (
-                              <span className="nowrap" key={id}>
-                                <SDKLanguageLogo language={id} size={14} />
-                                <span
-                                  className="text-muted font-weight-bold"
-                                  style={{ marginLeft: 2, verticalAlign: 3 }}
-                                >
-                                  {languageMapping[id].label}
-                                </span>
-                                {i <
-                                  selectedLanguagesWithoutRemoteEvalSupport.length -
-                                    1 && ", "}
-                              </span>
-                            )
-                          )}
-                        </div>
+                        <FaExclamationCircle /> Remote evaluation may not be
+                        available in your current SDK.
+                        {languages.length === 1 && (
```

Comment:
I noticed in a comment in another file you added a comment that we prefer to have a single language in this array. Do we surface that as a callout to the user anywhere in the UI? Might be out of scope, but just curious if we do since it seems like we want to steer them in that direction

Reply:
we basically force them to have array size of 1 for new SDK Connections, but we try not to force their hand with pre-existing connections. I don't think it needs to be called out anywhere since the UX kinds of pulls them towards 1.